GANDLF/cli/main_run.py

```diff
@@ -13,8 +13,15 @@
 
 
 def main_run(
-    data_csv, config_file, model_dir, train_mode, device, resume, reset, output_dir=None
-):
+    data_csv: str,
+    config_file: str,
+    model_dir: str,
+    train_mode: bool,
+    device: str,
+    resume: bool,
+    reset: bool,
+    output_dir: str = None,
```

Comment:
output_dir should be Union[str,None] in this case too

Reply:
Fixed with `Optional`.